src/components/MetadataForm/MetadataForm.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { ComponentProps } from 'react';
+import { MetadataFormStyle } from './MetadataFormStyles';
+import { JsonForms } from '@jsonforms/react';
+import MetadataColumnHeaders from '../MetadataColumnHeaders/MetadataColumnHeaders';
+import Form from '../Form/Form';
+import { OnChangeCallback } from '../../types';
+
+type Props = Omit<ComponentProps<typeof JsonForms>, 'onChange'> & {
+	onChange: OnChangeCallback;
+};
+
+const MetadataForm: React.FC<Props> = ({
+	ajv,
+	i18n,
+	schema,
+	uischema,
+	renderers,
+	data = {},
+	validationMode = 'ValidateAndShow',
+	onChange,
+}) => {
+	return (
+		<MetadataFormStyle>
+			<h2>Metadata toevoegen</h2>
+			<MetadataColumnHeaders
+				columns={[
+					{
+						header: 'Metadataveld',
+						width: 50,
+					},
+					{
+						header: 'Waarde',
+						width: 50,
```

Comment:
Hoeft dit nooit overschrijfbaar te zijn vanuit een conuming applicatie?
Waarom niet een child of ander passed component renderen met een default fallback naar dit specifieke component met deze config?

Reply:
It doesn't look we need that ATM. I consciously didn't go the extra mile. I figured we can always adjust this in the future if needed.